src/main/java/com/atlassian/tutorial/Service/AttachmentsService.java

```diff
@@ -0,0 +1,17 @@
+package com.atlassian.tutorial.Service;
+
+import com.atlassian.activeobjects.tx.Transactional;
+import com.atlassian.tutorial.entity.AttachmentsEntity;
+
+import java.sql.SQLException;
+import java.util.List;
+
+@Transactional
+public interface AttachmentsService
+{
```

Comment:
the brace ``public interface AttachmentsService{ ``should be on the same line as the class name. Use please java code style